### DISCUSSION TITLE
Launched Carawayhome.com with Stylex - Loving it ❤️

### DISCUSSION THREAD

**Discussion by TeoAlmonte** [facebook/stylex#816] [Show and tell]
Hey Everyone! I wanted to share that we launched our re-platformed site on November 1st using StyleX, and it’s been going well! 

Our team quickly adapted and was able to pick up the transition with ease given our team's strong JS-in-CSS knowledge. There was some initial resistance at first but once we got going the switch from Styled Components to Stylex-JS felt seamless. We've already noticed the performance improvements which have been fantastic, and we’ve been able to streamline both development and user experience.

Using Shopify’s Hydrogen-React,  Stylex-JS, Next.js and TypeScript, we were able to build a fast, flexible, and scalable site that supported record-breaking sales and increased engagement. We also introduced a cohesive style guide and design system to unify the experience (themes, themes, themes!!)

Check out the new site at [carawayhome.com](https://www.carawayhome.com/). If anyone has questions about our migration or the switch to Stylex-JS, feel free to ask—I’m happy to share insights!

**Comment by nmn** [comment]
What is your bundler setup like?

What are the biggest remaining pain points?
